src/components/voting/voting.js

```diff
@@ -175,19 +110,22 @@ class Voting extends React.Component {
               <TableCell>Uptime</TableCell>
               <TableCell>Approval</TableCell>
             </TableHead>
-            {this.state.delegates.map(item => (
+            {this.props.delegates.map(item => (
               <VotingRow key={item.address} data={item}
-                addToVoteList={this.props.addToVoteList}
-                removeFromVoteList={this.props.removeFromVoteList}
+                voteToggled={this.props.voteToggled}
+                voteStatus={this.props.votes[item.username]}
               />
             ))}
           </Table>
         </div>
-        {this.state.notFound}
+        {
+          (!this.isInitial && this.props.delegates.length === 0) &&
+          <div className='hasPaddingRow empty-message'>No delegates found</div>
+        }
         <Waypoint bottomOffset='-80%'
-                  scrollableAncestor={window}
-                  key={this.state.delegates.length}
-                  onEnter={this.loadMore.bind(this)}></Waypoint>
+          scrollableAncestor={window}
+          key={this.props.delegates.length}
+          onEnter={this.loadMore.bind(this)}></Waypoint>
```

Comment:
`loadMore` function doesn't work when page scroller rich end of the page

Reply:
Done